src/components/edit/AreaCRUD.tsx

```diff
@@ -4,9 +4,28 @@ import { AreaSummaryType } from '../crag/cragSummary'
 import { DeleteAreaTrigger, AddAreaTrigger, AddAreaTriggerButtonMd, AddAreaTriggerButtonSm, DeleteAreaTriggerButtonSm } from './Triggers'
 import { AreaEntityIcon } from '../EntityIcons'
 import NetworkSquareIcon from '../../assets/icons/network-square-icon.svg'
+import React, { useState } from 'react'
+import useUpdateAreasCmd from '../../js/hooks/useUpdateAreasCmd'
+import { useSession } from 'next-auth/react'
```

Comment:
Not super important. I usually place library imports at the top and then our own stuff.  `ts-standard` doesn't enforce this rule but I think it helps you quickly gain some context when scanning the import section.

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

Reply:
Fixed!